packages/js-sdk/src/datastore/index.ts

```diff
@@ -1,17 +1,20 @@
 import isString from 'lodash/isString';
 import { KinveyError } from '../errors/kinvey';
+import { logger } from '../log';
 import { isValidTag } from './cache';
 import { NetworkStore } from './networkstore';
 import { CacheStore } from './cachestore';
 import { DataStoreCache } from './cache';
+import { AutoStore } from './autostore';
 
 export enum DataStoreType {
+  Auto = 'Auto',
   Cache = 'Cache',
   Network = 'Network',
   Sync = 'Sync'
 };
 
-export function collection(collectionName: string, type = DataStoreType.Cache, options: any = {}) {
```

Comment:
Is there a way to deprecate this `collection` function? If not, the default DataStoreType should remain `Cache`, to avoid making this release a breaking change.

Reply:
The default datastore type is not `Auto` and not `Cache`.

Reply:
As per our conversation, please revert this to avoid the breaking change for now.

Reply:
This was reverted with https://github.com/Kinvey/js-sdk/pull/409/commits/23e80c88e8c118efb2c95e7c85bb2c243f0b08a6